### PR TITLE
The function formerly known as func_80282E30

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀 This PR matches 1 function (+0.02%) on `us`.

File: src/world/script_api/push_blocks.c [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 extern PushBlockGrid* wPushBlockGrids[8];
 
-// outVars for values returned by FetchPushedBlockProperties (func_80282E30)
+// outVars for values returned by FetchPushedBlockProperties
 enum {
     BLOCK_PROP_X    = LVar0,
```
Comment (HailSanta) [inline src/world/script_api/push_blocks.c:8]: Ah, I see now these are backwards. X/Y/Z are on LVars 3-5 and I/J/K are on LVars 0-2.

File: src/world/script_api/push_blocks.c [hunk 2]
```diff
@@ -157,13 +148,12 @@ API_CALLABLE(func_80282E30) {
     yThing = gPlayerStatus.position.y;
     zThing = gPlayerStatus.position.z;
 
-    xThing = xThing - x;
-    yThing = yThing - y;
-    zThing = zThing - z;
+    xThing -= x;
```
Comment (HailSanta) [inline src/world/script_api/push_blocks.c:151]: Could make these relativeX/Y/Z, although they are eventually the grid indices.

File: src/world/script_api/push_blocks.c [hunk 3]
```diff
@@ -129,25 +129,16 @@ API_CALLABLE(FinishPushBlockMotion) {
...
-//TODO FetchPushedBlockProperties
-API_CALLABLE(func_80282E30);
-// regalloc
-#ifdef NON_MATCHING
-API_CALLABLE(func_80282E30) {
+API_CALLABLE(FetchPushedBlockProperties) {
     PushBlockGrid* grid = (PushBlockGrid*) script->varTable[10];
     Entity* entity = get_entity_by_index(script->varTable[11]);
-    s32 xThing, yThing, zThing;
     s32 x, y, z;
```
Comment (HailSanta) [inline src/world/script_api/push_blocks.c:135]: centerX/Y/Z

**Review by HailSanta — APPROVED** [review]

**Pull request merged**